CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.3.4
```

Comment:
I saw your comment regarding release, but its not a problem. Next release is 1.3.3 and i can resolve merge conflicts by hand. Its also just important that the maintainers know what is current release and advise contributors to either wait or adjust release version (including since tags of course)

👍

Reply:
Did the merge